lobby.go

```diff
@@ -207,6 +207,50 @@ func wsListen(lobby *Lobby, player *Player, socket *websocket.Conn) {
 						lobby.Drawer.WriteAsJSON(JSEvent{Type: "your-turn"})
 					}
 				}
+			} else if received.Type == "on-kick-button" {
+				toKickID, isString := (received.Data).(string)
+				toKick := -1
+				if !isString {
+					fmt.Println("Invalid data")
+					continue
+				}
+
+				for index, otherPlayer := range lobby.Players {
+					if otherPlayer.ID == toKickID {
+						toKick = index
+					}
+				}
+
+				if toKick != -1 {
+					if player.votedForKick[toKickID] {
+						continue
+					}
+
+					player.votedForKick[toKickID] = true
+					playerToKick := lobby.Players[toKick]
+					playerToKick.voteKickCount++
+
+					if playerToKick.voteKickCount > 2 {
+						if lobby.Drawer == playerToKick {
+							endRound(lobby)
+						}
+						isPlayerToKickTheLobbyOwner := (lobby.Owner == playerToKick)
+						player.votedForKick[toKickID] = false
```

Comment:
You only reset this for the last player that voted, the others will still have the trash in their `voteForKick` map.

Reply:
You could also remove the entry instead of setting it to false: `remove(player.votedForKick, toKickID)`